ec2listimg

```diff
@@ -0,0 +1,192 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ec2publishimg. If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import os
+import sys
+
+
+import ec2imgutils.ec2utils as utils
+import ec2imgutils.ec2listimg as ec2lsimg
+from ec2imgutils.ec2imgutilsExceptions import (
+    EC2AccountException,
+    EC2ListImgException
+)
+
+# Set up command line argument parsing
+argparse = argparse.ArgumentParser(description='List account owned images')
+argparse.add_argument(
+    '-a', '--account',
+    dest='account_name',
+    help='Account to use',
+    metavar='ACCOUNT_NAME',
+)
+argparse.add_argument(
+    '--access-id',
+    dest='access_key',
+    help='AWS access key (Optional)',
+    metavar='AWS_ACCESS_KEY'
+)
+argparse.add_argument(
+    '-f', '--file',
+    default=os.sep.join(['~', '.ec2utils.conf']),
+    dest='config_file_path',
+    help='Path to configuration file, default ~/.ec2utils.conf (Optional)',
+    metavar='CONFIG_FILE'
+)
+publish_image_condition_group = argparse.add_mutually_exclusive_group()
+publish_image_condition_group.add_argument(
+    '--image-id',
+    dest='image_id',
+    help='The AMI ID of the image to be published (Optional)',
+    metavar='AMI_ID'
+)
+publish_image_condition_group.add_argument(
+    '--image-name',
+    dest='image_name',
+    help='The image name of the image to be published (Optional)',
+    metavar='IMAGE_NAME'
+)
+help_msg = 'An image name fragment to match the image name of the image to be '
+help_msg += 'published (Optional)'
+publish_image_condition_group.add_argument(
+    '--image-name-frag',
+    dest='image_name_frag',
+    help=help_msg,
+    metavar='IMAGE_NAME_FRAGMENT'
+)
+help_msg = 'A regular expression to match the image name of the image to be '
+help_msg += 'published (Optional)'
+publish_image_condition_group.add_argument(
+    '--image-name-match',
+    dest='image_name_match',
+    help=help_msg,
+    metavar='REGEX'
+)
+help_msg = 'Comma separated list of regions for publishing, all integrated '
+help_msg += 'regions if not given (Optional)'
+argparse.add_argument(
+    '-r', '--regions',
+    dest='regions',
+    help=help_msg,
+    metavar='EC2_REGIONS'
+)
+argparse.add_argument(
+    '-s', '--secret-key',
+    dest='secret_key',
+    help='AWS secret access key (Optional)',
+    metavar='AWS_SECRET_KEY'
+)
+help_msg = 'Set vebosety level 0 (default) image name, 1 image name and ID, '
+help_msg += '2 full description'
+argparse.add_argument(
+    '--verbose',
+    type=int,
+    default=0,
+    dest='verbose',
+)
+argparse.add_argument(
+    '--version',
+    action='store_true',
+    default=False,
+    dest='version',
+    help='Program version'
+)
+
+args = argparse.parse_args()
+
+if args.version:
+    version_file_name = 'VERSION'
+    base_path = os.path.dirname(utils.__file__)
+    version = open(base_path + os.sep + version_file_name, 'r').read()
```

Comment:
Nitpick: it's nice to strip the newline before output otherwise the output ends up like 'version'\n\n.